jetty-core/jetty-deploy/src/main/java/org/eclipse/jetty/deploy/providers/ContextProvider.java

```diff
@@ -144,6 +146,31 @@ public Map<String, String> getProperties()
         return _properties;
     }
 
+    public void loadProperties(Resource resource) throws IOException
+    {
+        Properties props = new Properties();
+        try (InputStream inputStream = resource.newInputStream())
+        {
+            props.load(inputStream);
+            props.forEach((key, value) -> _properties.put((String)key, (String)value));
+        }
+    }
+
+    public void loadPropertiesFromPath(Path path) throws IOException
+    {
+        Properties props = new Properties();
+        try (InputStream inputStream = Files.newInputStream(path))
+        {
+            props.load(inputStream);
+            props.forEach((key, value) -> _properties.put((String)key, (String)value));
+        }
+    }
+
+    public void loadPropertiesFromString(String path) throws IOException
```

Comment:
These 3 replace the old `org.eclipse.jetty.deploy.PropertiesConfigurationManager` functionality (that was removed in Jetty 12)